### PR TITLE
Specify that user/password is required for dicts in SQL Console

### PR DESCRIPTION
**Pull request by justindeguzman**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

@DanRoscigno @rfraposa for review

### CODE REVIEW

**Comment by robot-clickhouse-ci-2** [comment]
This is an automated comment for commit 28703ccead04e201b2ca6dd4ae3f64472e648de4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51196/28703ccead04e201b2ca6dd4ae3f64472e648de4/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: docs/en/sql-reference/statements/create/dictionary.md [hunk 1]
```diff
@@ -82,6 +82,28 @@ LIFETIME(MIN 0 MAX 1000)
 LAYOUT(FLAT())
 ```
 
+:::note
+When using the SQL console in [ClickHouse Cloud](https://clickhouse.com), you must specify a user (`default` or any other user) and password when creating a dictionary.
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/create/dictionary.md:86]:
```suggestion
When using the SQL console in [ClickHouse Cloud](https://clickhouse.com), you must specify a user (`default` or any other user with the role `default_role`) and password when creating a dictionary.
```

File: docs/en/sql-reference/statements/create/dictionary.md [hunk 2]
```diff
@@ -82,6 +82,28 @@ LIFETIME(MIN 0 MAX 1000)
...
+:::note
+
+```sql
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/create/dictionary.md:89]:
```suggestion
```sql
CREATE USER IF NOT EXISTS clickhouse_admin
IDENTIFIED WITH sha256_password BY 'passworD43$x';

GRANT default_role TO clickhouse_admin;

```

File: docs/en/sql-reference/statements/create/dictionary.md [hunk 3]
```diff
@@ -82,6 +82,28 @@ LIFETIME(MIN 0 MAX 1000)
...
+) ENGINE = MergeTree
+PRIMARY KEY id;
+
+CREATE DICTIONARY foo_db.id_value_dictionary
+(
+    id UInt64,
+    value String
+)
+PRIMARY KEY id
+SOURCE(CLICKHOUSE(TABLE 'source_table' USER 'foo_user' PASSWORD 'foo_user_complex_password' DB 'foo_db' ))
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/create/dictionary.md:102]:
```suggestion
SOURCE(CLICKHOUSE(TABLE 'source_table' USER 'clickhouse_admin' PASSWORD 'passworD43$x' DB 'foo_db' ))
```

File: docs/en/sql-reference/statements/create/dictionary.md [hunk 4]
```diff
@@ -82,6 +82,28 @@ LIFETIME(MIN 0 MAX 1000)
 LAYOUT(FLAT())
 ```
 
+:::note
+When using the SQL console in [ClickHouse Cloud](https://clickhouse.com), you must specify a user (`default` or any other user) and password when creating a dictionary.
+:::note
+
+```sql
+CREATE TABLE foo_db.source_table (
```
Comment (DanRoscigno) [inline docs/en/sql-reference/statements/create/dictionary.md:90]:
```suggestion
CREATE DATABASE foo_db;

CREATE TABLE foo_db.source_table (
```

**Review by DanRoscigno — CHANGES_REQUESTED** [review]
Thanks.

**Review by DanRoscigno — APPROVED** [review]

**Pull request merged**